docs/install.rst

```diff
@@ -29,8 +29,8 @@ To allow the tests to run actual queries against Copernicus Open Access Hub set
     export DHUS_USER=<your scihub username>
     export DHUS_PASSWORD=<your scihub password>
 
-and add ``--vcr disable`` to ``py.test`` arguments.
-To update the recordings use either ``--vcr record_new`` or ``--vcr reset``.
+and add ``--disable-vcr`` to ``py.test`` arguments.
```

Comment:
should also be ``pytest`` for consistency

Reply:
fixed, thanks!